src/BenchmarkDotNet.Diagnostics.Windows/EtwDiagnoser.cs

```diff
@@ -76,6 +78,8 @@ protected void Stop()
             WaitForDelayedEvents();
 
             Session.Dispose();
+
+            Console.CancelKeyPress -= OnConsoleCancelKeyPress;
```

Comment:
>CancelKeyPress [](start = 20, length = 14)

This event will only handle Ctrl-C and Ctrl-Break signals. It will not dispose the session if the cmd window is closed.